Spectrum-Analyzer/Spectrum-Analyzer/Src/MVC/Cpp/SpectrumController.cpp

```diff
@@ -81,7 +78,7 @@ void SpectrumController::handleMouseClick(GLFWwindow* window, int button, int ac
                 IModel->setCurrentListMenuOption(clickedPlot->getMethodFlag());
             }
             else {
-                cout << "ERROR IN SpectrumController::handleMouseClick.     IModel->getCurrentListMenuID() not a valid ID.";
+                std::cout << "ERROR IN SpectrumController::handleMouseClick.     IModel->getCurrentListMenuID() not a valid ID.";
```

Comment:
this string is long - can we define these errors somewhere?

Reply:
No